src/runtime/Temporal.h

```diff
@@ -39,6 +39,18 @@ class Temporal : public DerivedObject {
         return true;
     }
 
+    static Value getOptionsObject(ExecutionState& state, const Value& options)
+    {
+        if (options.isUndefined()) {
+            return new Object(state, Object::PrototypeIsNull);
```

Comment:
Do we need to create an object here? I usually prefer a constant such as null/undefined here.